server/models/plugins/cardCheckListPlugin.js

```diff
@@ -0,0 +1,59 @@
+const { Schema } = require("mongoose");
+const BasePlugin = require("./BasePlugin");
+
+const CheckListSchema = new Schema({
+  checklist: [{ item: String, completed: Boolean }],
+});
+
+const CardChecklistPluginSchema = BasePlugin.discriminator(
+  "CardCheckListPlugin",
+  new Schema(
+    {
+      cardCheckList: [CheckListSchema],
+    },
+    { timestamps: true }
+  )
+);
+
+CardChecklistPluginSchema.methods.get = async (pluginId) => {
+  const plugin = await this.findById(pluginId);
+  return plugin.checklist;
+};
```

Comment:
We'll verify the plugin exists through middleware this should only be manipulating data we shouldn't have to query again. 🤔

Reply:
Also, we won't be passing a pluginId to the arguments it will be a data object `{ body, params, query }`

We should also be sending a generalized response in all action handlers. `{ response: XXX, data or message: XYZ }`

Reply:
Let's say all 2xx status codes will respond in the format of `{ response: 2xx, data: {} }`
And anything else will response `{ response: XXX, message: "message goes here" }`